UpdateProfile.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  </head>
+  <body>
+    <div class="container">
+      <div class="post-form">
+        <div class="form-row mb-lg-4">
+          	<img src="Jaime.jpg" class="rounded-circle" style="height:200px; margin-left:15px; padding: 10px;" />
+
+            <div class="form-column">
+              <div class="form-group form-inline">
+                <label for="exampleInputEmail1" class="col-lg-3 col-form-label" style="padding:10px">Name:</label>
+                <input type="text" class="form-control" id="Name" placeholder="Name">
+              </div>
+
+              <div class="form-group form-inline">
+                <label for="exampleInputEmail1" class="col-lg-3 col-form-label" style="padding:10px" >Contact Number:</label>
+                <input type="text" class="form-control" id="exampleInputEmail1" placeholder="0917XXXXX">
+              </div>
+            </div>
+            </div>
+
+            <div class="form-row mb-lg-5">
+        			<label for="facebook-name" class="col-lg-3 col-form-label">Additonal Details:</label>
+        			<div class="col-lg-3">
+        			</div>
+              <div class="col-lg-5">
+                  <label for="fadasdasdas-name" class="col-lg-3 col-form-label">Visibility</label>
+                </div>
+              </div>
+        		</div>
+
+
+
+
+            <div class="form-row mb-lg-4">
+        			<label for="facebook-name" class="col-lg-3 col-form-label">Facebook Name: </label>
+        			<div class="col-lg-3">
+        				<input type="text" class="form-control form-control-sm py-lg-3 px-lg-3" id="facebook-name">
+        			</div>
+
+              <div class="col-lg-5">
+                <div class="custom-control custom-switch">
+                  <input type="checkbox" class="custom-control-input" id="facebook-name-switch">
+                  <label class="custom-control-label" for="facebook-name-switch"></label>
+                </div>
+              </div>
+        		</div>
```

Comment:
You can decrease the spacing between the label and the input field a bit, since remember that this is gonna be in a modal so it'll be a smaller window. Then increase the spacing between the switch and the input field. You can do this by using the [offsetting of Bootstrap](https://getbootstrap.com/docs/4.0/layout/grid/#offsetting-columns).

Do this also for the other fields.